build.sh

```diff
@@ -4,15 +4,17 @@ set -euxo pipefail
 
 # ESM
 
-tsc --project ./tsconfig-esm.json
+./node_modules/.bin/tsc --project ./tsconfig-esm.json
```

Comment:
```suggestion
yarn tsc --project ./tsconfig-esm.json
```

Reply:
I go back to the original version. I was executing ./build.sh instead of yarn build. It works correctly when executing yarn build. So I won't apply the change.